profiling/profiling.go

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// The profiling package exposes methods to manage profiling within gRPC.
```

Comment:
This should be 
```golang
// Package profiling exposes methods to manage profiling within gRPC.
```
I'm pretty sure vet/lint would complain otherwise.

Reply:
Done.